e2e/babel-plugin-jest-hoist/__tests__/integration.test.js

```diff
@@ -34,6 +34,10 @@ let e;
   jest.unmock('../__test_modules__/e');
 })();
 
+// Variable names prefixed with `mock` (ignore case) should not throw as out-of-scope
```

Comment:
looking at https://github.com/facebook/jest/pull/9806/files#r415385512, I think the one you move should just be `g` instead? Not sure, looks like a half completed refactor? Would love @jeysal's comments on this

Reply:
Yeah I agree, we can use a different file and avoid inserting this in the middle of the 'These will all be hoisted above imports' cases